src/sysv/systemd/sssd.service.in

```diff
@@ -15,7 +15,7 @@ ExecStartPre=+-/bin/chown -f @SSSD_USER@:@SSSD_USER@ @sssdconfdir@/sssd.conf
 ExecStartPre=+-/bin/chown -f -R @SSSD_USER@:@SSSD_USER@ @sssdconfdir@/conf.d
 ExecStartPre=+-/bin/chown -f -R @SSSD_USER@:@SSSD_USER@ @sssdconfdir@/pki
 ExecStartPre=+-/bin/sh -c "/bin/chown -f @SSSD_USER@:@SSSD_USER@ @dbpath@/*.ldb"
-ExecStartPre=+-/bin/sh -c "/bin/chown -f @SSSD_USER@:@SSSD_USER@ @gpocachepath@/*"
+ExecStartPre=+-/bin/sh -c "/bin/chown -f -R @SSSD_USER@:@SSSD_USER@ @gpocachepath@/*"
```

Comment:
Hi,

do you think keeping the `*` is important? Without the `*` there would be no error if the directory is empty but we would change the owner of hidden files and directories as well.

bye,
Sumit

Reply:
Thank you, fixed.

Reply:
Thanks